sharepoint/sharepoint-ps/sharepoint-online/Start-SPODataAccessGovernanceInsight.md

```diff
@@ -0,0 +1,254 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+author: pullabhk
+ms.author: pullabhk
+manager: 
+ms.reviewer:
+---
+
+# Start-SPODataAccessGovernanceInsight
+
+## SYNOPSIS
+
+This cmdlet generates Data Access Governance (DAG) reports meant to provide insights into potential oversharing of sensitive data in SharePoint and/or OneDrive for Business. SharePoint Advanced Management (SAM) license is required to run these reports.
+
+## SYNTAX
+
+### EEEUParameterSet
+```
+Start-SPODataAccessGovernanceInsight -ReportEntity <ReportEntityEnum> -Workload <WorkloadEnum>
+ -ReportType <ReportTypeEnum> -Name <String>
+ [-Template <System.Collections.Generic.List`1[Microsoft.Online.SharePoint.TenantAdministration.TemplateEnum]>]
+ [-Privacy <PrivacyEnum>] [-SiteSensitivityLabelGUID <System.Collections.Generic.List`1[System.Guid]>]
+ [<CommonParameters>]
+```
+
+### SharingLinkParameterSet
+```
+Start-SPODataAccessGovernanceInsight -ReportEntity <ReportEntityEnum> -Workload <WorkloadEnum>
+ -ReportType <ReportTypeEnum> [<CommonParameters>]
+```
+
+### LabelParameterSet
+```
+Start-SPODataAccessGovernanceInsight -ReportEntity <ReportEntityEnum> -Workload <WorkloadEnum>
+ -ReportType <ReportTypeEnum> [-FileSensitivityLabelName <String>] -FileSensitivityLabelGUID <Guid>
+ [<CommonParameters>]
+```
+
+### SitePermissionsParameterSet
+```
+Start-SPODataAccessGovernanceInsight -ReportEntity <ReportEntityEnum> -Workload <WorkloadEnum>
+ -ReportType <ReportTypeEnum> -Name <String>
+ [-Template <System.Collections.Generic.List`1[Microsoft.Online.SharePoint.TenantAdministration.TemplateEnum]>]
+ [-Privacy <PrivacyEnum>] [-SiteSensitivityLabelGUID <System.Collections.Generic.List`1[System.Guid]>]
+ -CountOfUsersMoreThan <Int32> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet is used to generate Data Access Governance (DAG) reports which deal with potential oversharing of sensitive data. These reports are present in Sharepoint admin center. Reports are currently available for the following scenarios:
```

Comment:
>Data Access Governance (DAG) [](http://example.com/codeflow?start=32&length=28)

in this case you can just say "DAG" since you already defined it line 16